ext/opentelemetry-ext-django/tests/test_middleware.py

```diff
@@ -0,0 +1,13 @@
+from django.test import Client, SimpleTestCase
+
+from opentelemetry.test.wsgitestutil import WsgiTestBase
+
+
+class TestDjangoOpenTracingMiddleware(WsgiTestBase, SimpleTestCase):
```

Comment:
I don't know if you are planning it but I think it should have some more cases, like span attributes, name, etc.

Reply:
Yes, definitely.

Reply:
Added more test cases.